deltacat/compute/compactor_v2/utils/content_type_params.py

```diff
@@ -75,11 +76,16 @@ def _download_parquet_metadata_for_manifest_entry(
     entry_index: int,
     deltacat_storage: unimplemented_deltacat_storage,
     deltacat_storage_kwargs: Optional[Dict[Any, Any]] = {},
+    file_reader_kwargs_provider: Optional[ReadKwargsProvider] = None,
 ) -> Dict[str, Any]:
+    logger.info(
+        f"Downloading the parquet metadata for Delta with locator {delta.locator} and entry_index: {entry_index}"
+    )
     pq_file = deltacat_storage.download_delta_manifest_entry(
         delta,
         entry_index=entry_index,
         table_type=TableType.PYARROW_PARQUET,
+        file_reader_kwargs_provider=file_reader_kwargs_provider,
```

Comment:
should we remove `file_reader_kwargs_provider` from `deltacat_storage_kwargs` to prevent any conflicts here?

Reply:
Should we check and pop it out? That seems reasonable

Reply:
Yeah.

Reply:
Done